.github/workflows/py-build-main.yml

```
@@ -132,7 +134,4 @@ jobs:
           path: dist
           merge-multiple: true
 
-      - uses: pypa/gh-action-pypi-publish@v1.4.2
-        with:
-          user: aicspypi
-          password: ${{ secrets.PYPI_TOKEN }}
+      - uses: pypa/gh-action-pypi-publish@release/v1
```

```suggestion
      - uses: pypa/gh-action-pypi-publish@ec4db0b4ddc65acdf4bff5fa45ac92d78b56bdf0
```

That's the commit SHA of the latest version ([v1.9.0](https://github.com/pypa/gh-action-pypi-publish/releases/tag/v1.9.0)) and preferred to use that because:

> [Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions)